src/Symfony/Bundle/FrameworkBundle/Resources/config/routing.php

```diff
@@ -0,0 +1,170 @@
+use Symfony\Bundle\FrameworkBundle\Routing\RedirectableCompiledUrlMatcher;
+use Symfony\Bundle\FrameworkBundle\Routing\Router;
+use Symfony\Component\Config\Loader\LoaderResolver;
+use Symfony\Component\HttpKernel\EventListener\RouterListener;
+use Symfony\Component\Routing\Generator\CompiledUrlGenerator;
+use Symfony\Component\Routing\Generator\Dumper\CompiledUrlGeneratorDumper;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\Loader\ContainerLoader;
+use Symfony\Component\Routing\Loader\DirectoryLoader;
+use Symfony\Component\Routing\Loader\GlobFileLoader;
+use Symfony\Component\Routing\Loader\PhpFileLoader;
+use Symfony\Component\Routing\Loader\XmlFileLoader;
+use Symfony\Component\Routing\Loader\YamlFileLoader;
+use Symfony\Component\Routing\Matcher\Dumper\CompiledUrlMatcherDumper;
+use Symfony\Component\Routing\Matcher\ExpressionLanguageProvider;
+use Symfony\Component\Routing\Matcher\UrlMatcherInterface;
+use Symfony\Component\Routing\RequestContext;
+use Symfony\Component\Routing\RequestContextAwareInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+return static function (ContainerConfigurator $container) {
+    $container->parameters()->set('router.request_context.host', 'localhost');
+    $container->parameters()->set('router.request_context.scheme', 'http');
+    $container->parameters()->set('router.request_context.base_url', '');
+
+    $container->services()
+        ->set('routing.resolver', LoaderResolver::class)
+
+        ->set('routing.loader.xml', XmlFileLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader.yml', YamlFileLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader.php', PhpFileLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader.glob', GlobFileLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader.directory', DirectoryLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader.container', ContainerLoader::class)
+            ->tag('routing.loader')
+            ->args([
+                service('file_locator'),
+            ])
+
+        ->set('routing.loader', DelegatingLoader::class)
+            ->public()
+            ->args([
+                service('routing.resolver'),
+                [], // Default options
+                [], // Default requirements
+            ])
+
+        ->set('router.default', Router::class)
+            ->tag('monolog.logger', ['channel' => 'router'])
```

Comment:
please move `tag` after `args` and `call`

Reply:
Moved down all the tags.